src/components/Canvas/Editor.tsx

```diff
@@ -1386,6 +1388,54 @@ export default class Editor extends EventDispatcher {
     }
   }
 
+  fillMissingIndices(previousCoord: Coord, currentCoord: Coord) {
```

Comment:
Great work. Would it be possible to move this function to the utils folder? This function does not necessarily need to be in the Editor component. We could make a function named `getInBetweenPixelIndices` that returns an array of pixel indices that should be colored. Based on the returned array, we can call the this.drawPixelInInteractionLayer.